### PR TITLE
Fix 6631  DragDrop Tree  keyboard navigation

### LINKED ISSUE #6631: Tree Node Drag & Drop Keyboard Support
I faced an issue when the Tree items became draggable, the keyboard keys were not working well (up, down, left, right, space, enter)
Can anyone no why? is it something that I'm doing wrong? I see it's either not working in the website example ([primereact.org/tree/#dragdrop](https://primereact.org/tree/#dragdrop))

https://github.com/orgs/primefaces/discussions/1883

### PR DESCRIPTION
**Pull request by KirilCycle**
### Defect Fixes
Fix #6631 Skip drag points where needed

### CODE REVIEW

**Comment by KirilCycle** [comment]
I've seen that primang Tree stores nodes and droppoints together in a special p-treenode tag, I don't know if this is the only point of having an additional tag, or if I should study their example deeper and try something similar, or if we are good with this solution ?

File: components/lib/tree/UITreeNode.js [hunk 1]
```diff
@@ -84,17 +84,41 @@ export const UITreeNode = React.memo((props) => {
         }
     };
 
+    const findNextNonDroppointSibling = (nodeElement) => {
```
Comment (KirilCycle) [inline components/lib/tree/UITreeNode.js:87]: conditions can be refactored

Comment (melloware) [reply components/lib/tree/UITreeNode.js:87]: I need to study this one...

**Review by nitrogenous — APPROVED** [review]
LGTM and Works Fine @gucal @melloware what do you people say?

**Review by melloware — APPROVED** [review]

**Comment by melloware** [comment]
Its good for me i was gonna refactor it slightly but i can do it after the PR is merged!

**Pull request merged**